plaso/multi_processing/task_engine.py

```diff
@@ -439,9 +441,10 @@ def _ScheduleTasks(self, storage_writer):
 
         if task:
           if self._ScheduleTask(task):
+            task_path_spec = task.path_spec.comparable.replace('\n', ' ')
```

Comment:
task_path_spec => task_path_spec_string or something to indicate this is not a path specification object but a string representation